<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate random password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>F:/Загрузки браузер/key.png</normaloff>F:/Загрузки браузер/key.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="generatePassBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Generate password</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="OutputPassword"/>
      </item>
      <item>
       <widget class="QPushButton" name="Generate_Password">
        <property name="text">
         <string>Generate password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="quantityBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLineEdit" name="QuantityPassSymbol">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QSlider" name="QuantityPassSymbolSlide">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="QuantitySettings">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="upperCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>123</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use uppercase letter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="numberCheck">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>85</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use numbers</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>F:/Загрузки браузер/lock.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
